kernel/patches/virtio-fs-v0.3.x/0001-arm64-mm-Enable-memory-hot-remove.patch

```diff
@@ -0,0 +1,454 @@
+From af857648ba026b21e04f2524a99cd311b5bc0243 Mon Sep 17 00:00:00 2001
```

Comment:
Is this patch really ~19 years old?

Reply:
Hi @jodh-intel, I don't think so. I just download this patch from linux patchwork and apply it. I think it's better to remove this line.

Reply:
Well, I think we need all the patches to be "pristine" / vanilla, but it looks very odd as-is...

Reply:
em, do you mean keep that line?  but I can't explain where that line come from.

Reply:
@jongwu - all I mean is, if it's part of the original patch, we should keep it (even if it is "wrong"). Can you provide the URL for the patch on patchwork?

Reply:
@jodh-intel , the URL is https://patchwork.kernel.org/cover/11127621/. I download the patch by click the "series" button. when I apply this patch, that line doesn't exist but will show when format it. it's strange and I think that line does not belong to the patch.

Reply:
Ok, thanks for the reference @jongwu.